### DISCUSSION TITLE
[Feature] Add option or change how selecting a photo(s) works to prevent unintended selection

### DISCUSSION THREAD

**Discussion by greggitter** [immich-app/immich#14932] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As a relatively new user, this could be my problem, but when browsing photos in Immich on an iPad or iPhone, I am constantly accidentally selecting images as I scroll and have to "x" out of that each time.  It just seems to easy to accidentally select image(s).  Pinching to shrink the thumbnail size of what's displayed also selects several images unintentionally.  Again, must then click "x" to cancel selection.

It seems like an alternative approach might be to have a selection button at the top like other apps where you are able to enter "selection mode", select what you want, choose the  operation (i.e., delete, etc) and then exit out of selection mode.

Maybe this exists already but I didn't see it in preferences or in the discussions or issues.

Awesome app, hoping to become a long-term user!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by smileBeda** [comment]
https://github.com/immich-app/immich/discussions/26432 was closed in favor of this older request.
